tests/security/postgresql_ssl/postgresql_ssl_client.pm

```diff
@@ -25,7 +25,7 @@ sub run {
     # We don't run setup_multimachine in s390x, but we need to know the server and client's
     # ip address, so we add a known ip to NETDEV
     my $netdev = get_var('NETDEV', 'eth0');
-    assert_script_run("ip addr add $server_ip/24 dev $netdev") if (is_s390x);
+    assert_script_run("ip addr add $client_ip/24 dev $netdev") if (is_s390x);
```

Comment:
not sure is the $server_ip wrong in previous PR ?

Reply:
Seems like I made a copy-paste error 😮.

Reply:
It indeed configure client IP address.

Reply:
good to know, thanks :)